code/script.js

```diff
@@ -1,51 +1,246 @@
+};
+
+const ticketPrices = {  // in us$
+  adultPrice: [11],
+  studentPrice: [8],
+  childPrice: [4],
+};
+
 
+/* ----- Declare your functions after this comment ----- */
 
-// Declare your functions after this comment
+// Clears the input field.
+const clearInputField = () => {
+  inputField.value = "";
+};
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// Removes all child elements from a specified parent element.
+function removeChildrenFunction() {
+  while (inputWrapper.hasChildNodes()) {
+    inputWrapper.removeChild(inputWrapper.firstChild);
+  }
+};
+
+// Refreshes the page.
+const reloadPage = () => {
+  setTimeout(() => {
+    location.reload();
+  }, 3000);
+};
+
+// Chat sound.
+const playAudio = () => {
+  const notification = document.getElementById('notification');
+  notification.volume = 0.6; // Set the volume to 60%
+  notification.play();
+};
+
+// Will add a chat bubble in the correct place. ShowMessage function takes arguments "message" and "sender".
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
+  playAudio();
+  // Checks if the sender is 'user' and if that's the case it inserts an html section inside the chat with the posted message.
   if (sender === 'user') {
+    console.log(message)
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user-icon.png" alt="User" />  
       </section>
-    `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
+    `;
+    // Checks if the sender is a 'bot' and if that's the case it inserts an html section inside the chat with the posted message.
   } else if (sender === 'bot') {
+    console.log(message)
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot-icon.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+    `;
+  };
+  
+  // Makes the chat scroll to the last message when there are too many in the chat box.
   chat.scrollTop = chat.scrollHeight
+};
+
+// Bot pop-up-function.
+// Adds a class to hide ticket-btn-container + removes class from chatbot so it's visible.
+const botPopUp = () => {
+  ticketBtnContainer.classList.add("ticketBtnContainer");
+  chatbot.classList.remove("chatbot");
+  setTimeout(() => greetUser(), 500);
 }
 
-// Starts here
+ticketBtn.addEventListener("click", botPopUp);
```

Comment:
I reeeeeally like the ticket-button that is displayed when the website is loaded. And that the bot doesn't appear until the the user has pressed the button. Very neat!